### PR TITLE
Update ClientCommands.md

### PR DESCRIPTION
**Pull request by DestinyDragon49**
Spanish characters corrected

### CODE REVIEW

**Comment by DestinyDragon49** [comment]
@Southclaws

File: docs/translations/es/client/ClientCommands.md [hunk 1]
```diff
@@ -5,19 +5,19 @@ descripion: Una lista de todos los comandos del cliente.
...
 | /audiomsg      | Activa y desactiva los mensajes que se imprimen cuando una URL de audio es transmitida al cliente.                                                                                                                                                                                                                                         |
-| /fontsize      | Cambia el tama�o del texto del UI (chats, di�logos, etc�tera). Valores v�lidos de -3 a 5.                                                                                                                                                                                                                                     |
+| /fontsize      | Cambia el tama&#241;o del texto del UI (chats, di&#225;logos, etc&#233;tera). Valores v&#225;lidos de -3 a 5.                                                                                                                                                                                                                                     |
```
Comment (FreddieCrew) [inline docs/translations/es/client/ClientCommands.md:20]:
```suggestion
| /fontsize      | Cambia el tamaño del texto del interfaz de usuario (chats, diálogos, etcétera). Valores válidos de -3 a 5.                                                                                                                                                                                                                             |
```

File: docs/translations/es/client/ClientCommands.md [hunk 2]
```diff
@@ -5,19 +5,19 @@ descripion: Una lista de todos los comandos del cliente.
...
 | /ctd           | Este comando fue agregando en SA:MP 0.3.7 RC2. It enables client debugging of the player camera target.                                                                                                                                                                                                                  |
```
Comment (FreddieCrew) [inline docs/translations/es/client/ClientCommands.md:21]:
```suggestion
| /ctd           | Este comando fue agregado en SA:MP 0.3.7 RC2. Habilita la depuración del objetivo de la cámara del jugador.                                                                                                                                                                                                                  |
```

File: docs/translations/es/client/ClientCommands.md [hunk 3]
```diff
@@ -5,19 +5,19 @@ descripion: Una lista de todos los comandos del cliente.
...
-| /rcon          | Mas relacionado al servidor que al cliente. Este comando es usado para ejecutar comandos RCON. RCON es el sistema de administraci�n integrado. RCON significa [Remote Control](../server/ControllingServer#using-rcon).                                                                                                      |
-| /hudscalefix   | Arregla la escala del HUD basado en la resoluci�n de pantalla del cliente.                                                                                                                                                                                                                                                               |
+| /rcon          | Mas relacionado al servidor que al cliente. Este comando es usado para ejecutar comandos RCON. RCON es el sistema de administraci&#243;n integrado. RCON significa [Remote Control](../server/ControllingServer#using-rcon).                                                                                                      |
```
Comment (FreddieCrew) [inline docs/translations/es/client/ClientCommands.md:22]:
```suggestion
| /rcon          | Más relacionado al servidor que al cliente. Este comando se usa para ejecutar comandos RCON. RCON es el sistema de administración integrado. RCON significa [Remote Control](../server/ControllingServer#using-rcon).                                                                                                   |
```

**Review by FreddieCrew — CHANGES_REQUESTED** [review]
You need to fix these issues before merging @DestinyDragon49

**Comment by adib-yg** [comment]
@FreddieCrew I don't think your changes will fix the unicode error. It's still the same.

image

**Comment by FreddieCrew** [comment]
@adib-yg I just tried it on my localhost, and it seems to be working perfectly fine.

screenshot

**Pull request merged**